paper/tex/discussion.tex

```diff
@@ -1,43 +1,113 @@
 
-\subsection{Future work}
-\label{sec:futurework}
-The work presented in this paper is only the first step in characterizing current photo-$z$ codes and moving toward an improved photometric redshift estimator.  This initial paper explored code performance in idealized conditions with perfect catalog-based photometry and representative training data.  As mentioned in Section~\ref{sec:stackedmetrics} for the stacked $N(z)$ metrics we examined only the entire galaxy population with no selections in either photo-$z$ ``quality" or redshift.  The cosmological analyses for weak lensing and large scale structure based measures plan to break galaxy samples into tomographic redshift bins, using photo-$z$ $p(z)$ to infer the redshift distribution for each bin.  The specific selection used to determine these bins, both algorithmically and the specific bin boundaries, could induce biases due to indirect selections inherent in the photo-$z$ or other bin selection parameters.  The effects of tomographic bin selection will be explored in a dedicated future paper, including propagation of redshift uncertainties in a set of fiducial tomographic redshift bins in order to estimate impact on cosmological parameter estimation.
+\subsection{Extensions to the experimental design}
+\label{sec:futureexperiments}
+
+%\begin{enumerate}
+%\item \red{the combination of codes}
+%\item \red{$p(z,\alpha)$}
+%\item \red{tomographic analysis later on}
+%\red{this paper deals with training, future papers will also explore calibration via cross-correlation methods}
+%\end{enumerate}
+%\red{mention the SRM once again, say that the plans are extensive, but we do have a plan and a rough timeline.}
+
+The work presented in this paper is only a first step in assessing \pzpdf\ approaches and moving toward an improved photometric redshift estimator.
+Here we discuss the next steps for subsequent investigations.
+
+This initial paper explored code performance in idealized conditions with perfect catalog-based photometry and representative training data.
+A top priority for a follow-up study is to test realistic forms of incomplete, erroneous, and non-representative template libraries and training sets as well as the impact of other forms of external priors that must be ingested by the codes, major concerns in \citet{Newman:2015, Masters:2017}.
+We plan to perform a full sensitivity analysis on a realistically incomplete training set of spectroscopic galaxies, modeling the performance of spectrographs, emission-line properties, and expected signal-to-noise to determine which potential training set galaxies are most likely to be excluded.
+In addition to outright redshift failures we will model the inclusion of a small number of high-confidence yet false redshifts due to emission line misidentification or noise spikes.
```

Comment:
I think we should add a note that codes that perform well in the presence of representative data may not perform as well when incompleteness is introduced to the training data, and future studies will examine which algorithms most effectively deal with this.